orchestrator-salt/src/main/resources/salt/pillar/postgresql/disaster_recovery.sls

```diff
@@ -0,0 +1,3 @@
+disaster_recovery:
+  object_storage_url:
```

Comment:
The value of this variable will be passed in a later commit?

Reply:
Yes, the intent is that we'll pass in a different value for the `object_storage_url` with each backup request. It was my impression that we could provide a different Pillar value per invocation, something like [this](https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/container/postgres/PostgresConfigService.java#L34)